src/main/java/edu/hawaii/its/api/service/MemberService.java

```diff
@@ -0,0 +1,54 @@
+package edu.hawaii.its.api.service;
+
+import edu.hawaii.its.api.exception.InvalidGroupPathException;
+import edu.hawaii.its.api.wrapper.HasMemberCommand;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Service;
+
+@Service("memberService")
+
+/**
+ * MemberService checks if a UH affiliate is a member of a group or grouping.
+ */
+public class MemberService {
+    @Value("${groupings.api.grouping_admins}")
+    private String GROUPING_ADMINS;
+
+    @Autowired
+    private GroupPathService groupPathService;
+
+    @Autowired
+    private ExecutorService executor;
+
+    public boolean isAdmin(String uhIdentifier) {
+        return executor.execute(new HasMemberCommand(GROUPING_ADMINS, uhIdentifier)).getResultCode().equals("IS_MEMBER");
+    }
+
+    public boolean isMember(String path, String uhIdentifier) {
+        HasMemberCommand hasMemberCommand = new HasMemberCommand(path, uhIdentifier);
+        return hasMemberCommand.execute().getResultCode().equals("IS_MEMBER");
+    }
+
+    public boolean isIncludeMember(String groupingPath, String uhIdentifier) {
+        String includePath = groupPathService.getIncludeGroup(groupingPath);
+        if (includePath.equals("")) {
+            throw new InvalidGroupPathException(groupingPath);
+        }
+        return isGroupMember(includePath, uhIdentifier);
+    }
+
+    public boolean isExcludeMember(String groupingPath, String uhIdentifier) {
+        return isGroupMember(groupingPath + ":exclude", uhIdentifier);
+    }
+
+    public boolean isOwner(String groupingPath, String uhIdentifier) {
+        return isGroupMember(groupingPath + ":owners", uhIdentifier);
+    }
```

Comment:
Your isExcludeMember and isOwner functions may be missing the `throw new InvalidGroupPathException`. I am working on a ticket where I am planning on using this service; could you add a isBasisMember function? Thanks!

Reply:
I see, I would stick with MemberAttributeService in your implementation for now, these services I implemented are not ready to be used in working code.

Reply:
But thanks for bringing this up, "isIncludeMember()" is not correct.